src/s3ql/backends/swiftks.py

```diff
@@ -121,7 +121,7 @@ def _get_conn(self):
                 }
             }
 
-            auth_url_path = '/v3/auth/tokens'
+            auth_url_path = ('custom-auth-url' in self.options) or '/v3/auth/tokens'
```

Comment:
`'custom-auth-url' in self.options` only checks if the user provided the option. it does not return the provided value.
 
```suggestion
            if 'custom-auth-url' in self.options:
                auth_url_path = self.options['custom-auth-url']
            else:
                auth_url_path = '/v3/auth/tokens'
```

Reply:
Ohhh. That explains why I it didn't work when I tested it.... Thanks a lot!